src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
+    private static final Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        int colors = random.nextInt(Colors.values().length);
+        return Colors.values()[colors].name();
```

Comment:
It's good that you're using the `name()` method to get the string representation of the enum constants, which is the recommended approach according to the checklist.